Assets/Shopify/UIToolkit/Shops/Generic/Classes/GenericCartView.cs

```diff
@@ -0,0 +1,53 @@
+namespace Shopify.UIToolkit {
+    using UnityEngine;
+    using UnityEngine.UI;
+    using Shopify.Unity;
+    using Shopify.Unity.SDK;
+    using System.Collections.Generic;
+    using System;
+
+    public class GenericCartView : MonoBehaviour {
+        public IShop Shop;
```

Comment:
I dont think the Generic Cart View should talk to the shop via IShop, I think it should hold a more specific reference `GenericMultiProductShop` or alternatively talk through a new `IGenericShop` interface or something

Reply:
Agreed - I made the update so the shop invokes methods on the `GenericCartView` instance it has.